data/json/items/armor.json

```diff
@@ -6289,10 +6290,10 @@
         "id" : "fireman_belt",
         "name" : "firefighter belt",
         "volume" : 2,
-        "material_thickness" : 0,
+        "material_thickness" : 2,
```

Comment:
This comment is correct, making this attribute the equivalent of dead code, please revert it to 0.